onnxruntime/core/providers/openvino/ov_versions/data_ops.cc

```diff
@@ -158,7 +158,7 @@ std::vector<SupportedOp> supported_op_mode = {
     {"InstanceNormalization", V_2020_4, {"CPU", "GPU"}},
     {"HardSigmoid", V_2020_4, {"CPU", "GPU"}},
     {"HardMax", V_2022_1, {"CPU", "GPU"}},
-    {"HardSwish", V_2023_3, {"CPU", "GPU"}},
+    {"HardSwish", V_2025_2, {"CPU", "GPU"}},
```

Comment:
Could you help me understand why the version is required to be so high as 2025_2? As my verification, at least 2025_0 supports this op. Could the version be lower for better compatibility?

Reply:
OVEP is compliant with the latest three releases of OV. Validation is also performed for the same. It can be from 2025_0 if its supported from then.

Reply:
@preetha-intel is right lets support from the original version

Reply:
@Kotomi-Du  It can be from 2025_0 as it will get validated against all our models.  
@mingmingtasd As its supported from the latest version of OVEP which supports OV from 2025.0 we can stick with that as we can promote users to move to latest version of OV for better performance.

Reply:
update, please review

Reply:
LGTM! Thanks for the fix. @Kotomi-Du @sfatimar @preetha-intel